test/internal/helpers/Preference.java

```diff
@@ -0,0 +1,44 @@
+package internal.helpers;
+
+import static org.hamcrest.Matchers.equalTo;
+
+import net.sourceforge.kolmafia.preferences.Preferences;
+import org.hamcrest.FeatureMatcher;
+import org.hamcrest.Matcher;
+
+public class Preference {
+  public static Matcher<String> hasValue(Matcher<? super String> prefMatcher) {
```

Comment:
Yeah that makes sense. The idea of this API is that you would rarely use `has*Value`, only doing so when you need to check whether an integer value is within some bounds (or to avoid the usual float -> string stuff)